app/views/handlers/confirmation_email.py

```diff
@@ -92,21 +94,19 @@ def is_enabled(schema: QuestionnaireSchema):
 
 
 @dataclass
-class ConfirmationEmailFulfilmentRequest(FulfilmentRequest):
+class ConfirmationEmailTask:
     email_address: str
     session_data: SessionData
     schema: QuestionnaireSchema
 
-    def _payload(self) -> Mapping:
-        return {
-            "fulfilmentRequest": {
+    @property
+    def payload(self):
+        return json.dumps(
+            {
                 "email_address": self.email_address,
+                "personalisation": {"address": self.session_data.display_address},
```

Comment:
Given that `personalisation` is specific to the choice of email provider (gov notify), I think we should revert to the flat structure we had previously and do the work to construct the gov notify payload in the cloud task.

Reply:
Consumer has been updated to expect `display_address` in the payload https://github.com/ONSdigital/eq-submission-confirmation-consumer/pull/2/commits/6ce9def679f5d7b473a8a9c71ca7e6ee9039d0d9